recipes/aws-noaa-cdr-ndvi/recipe.py

```diff
@@ -0,0 +1,30 @@
+import os
+from os.path import join
+
+import s3fs
+
+from pangeo_forge_recipes.patterns import pattern_from_file_sequence
+from pangeo_forge_recipes.recipes.reference_hdf_zarr import HDFReferenceRecipe
+
+url_base = 's3://noaa-cdr-ndvi-pds/data/'
+
+file_list = []
+fs = s3fs.S3FileSystem(anon=True)
+
+
+def is_nc(x):
+    return x.endswith('.nc')
+
+
+def add_s3(x):
+    return 's3://' + x
+
+
+years_folders = fs.ls(join(url_base))
+years = list(map(lambda x: os.path.basename(x), years_folders))
+
+for year in years:
+    file_list += sorted(filter(is_nc, map(add_s3, fs.ls(join(url_base, str(year)), detail=False))))
+
+pattern = pattern_from_file_sequence(file_list, 'time', nitems_per_file=1)
```

Comment:
for each of your PRs @kathrynberger maybe double check that nitems is always 1? found this to be not the case for many datasets where the file contained an entire years worth of data, or 8 time points per day, etc.

Reply:
Thanks, that's the mistake I had made originally and had revised all of my remaining PRs to be updated appropriately (and why I appreciate a second set of eyes reviewing the PR internally before posting to `pange-forge`. 
This recipe does in fact have a time step, n = 1, as there is one directory per year, with one file per day (each file containing one time step = 1 day).  👍